ibm-messaging-mq-cloud-showcase-app/backend/Dockerfile

```diff
@@ -12,28 +12,87 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
+# This Dockerfile has two separate stages.
+#The first stage is builds the Node program.
+# The second stage is a runtime-only container that holds just the things we need to
+# execute the compiled program.
+
+###########################################################
+# This starts the BUILD phase
+###########################################################
 FROM ubuntu:20.04 as builder
```

Comment:
As we are building a node.js image, is there a node.js base image that would the need to install node etc. ?